.github/workflows/check.yml

```diff
@@ -59,3 +59,36 @@ jobs:
         - name: Run Cargo Tests
           run: |
             cargo test
+  test-coverage:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set-Up
+        run: sudo apt install -y cmake pkg-config libssl-dev git build-essential clang libclang-dev curl
+
+      - name: Install Rustup
+        run: |
+          curl https://sh.rustup.rs -sSf | sh -s -- -y
+          source ~/.cargo/env
+          rustup default stable
+          rustup update nightly
+          rustup update stable
+          rustup target add wasm32-unknown-unknown --toolchain nightly
+          rustup component add llvm-tools-preview
+          cargo install grcov
+
+      - name: Test coverage
+        run: |
+          export CARGO_INCREMENTAL=0
```

Comment:
The following commands are referenced:
https://github.com/paritytech/polkadot/blob/83e5955b83870c0b107af37934d9f8a4ecf2c63d/doc/testing.md#coverage